python/pyspark/ml/tuning.py

```diff
@@ -112,15 +198,24 @@ class CrossValidator(Estimator, HasSeed):
     >>> cvModel = cv.fit(dataset)
     >>> evaluator.evaluate(cvModel.transform(dataset))
     0.8333...
+    >>> cvPath = temp_path + "/cv"
+    >>> cv.save(cvPath)
+    >>> loadedCV = CrossValidator.load(cvPath)
+    >>> loadedCV.getEstimator().uid == cv.getEstimator().uid
```

Comment:
I would recommend not bothering with these comparisons in the doctest since it's part of the documentation.  I would put them (and more) in a unit test.  Same for TrainValidationSplit